openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -642,6 +656,43 @@ const TeamDetailsV1 = ({
     [currentTeam.isJoinable]
   );
 
+  const DELETED_TEAMS_ITEM = [
+    {
+      label: (
+        <Space className="cursor-pointer manage-button" size={8}>
+          {deletedTeamIcon}
+          <div className="text-left" data-testid="deleted-team">
+            <Row className="m-b-xss" justify="space-between">
+              <Col>
+                <p className="font-medium" data-testid="deleted-label">
+                  {t('label.deleted-team-action', {
+                    action: showDeletedTeam ? t('label.hide') : t('label.show'),
+                  })}
+                </p>
+              </Col>
+
+              <Col>
+                <Switch
+                  checked={showDeletedTeam}
+                  className="m-r-xs"
+                  data-testid="show-deleted-switch"
+                  size="small"
+                  onChange={onShowDeletedTeamChange}
+                />
+              </Col>
+            </Row>
+            <Typography.Paragraph className="text-grey-muted text-xs m-b-0 line-height-16">
+              {t('label.view-deleted-teams')}
+            </Typography.Paragraph>
+          </div>
+        </Space>
+      ),
+      key: 'deleted-team-dropdown',
+    },
+  ];
+
+  const organizationDropdownContent = <Menu items={[...DELETED_TEAMS_ITEM]} />;
```

Comment:
```suggestion
  const organizationDropdownContent = <Menu items={DELETED_TEAMS_ITEM} />;
```

Avoid unnecessary spreading

Reply:
Now it will be `items={[DELETED_TOGGLE_MENU_ITEM]}`